commerce_coordinator/apps/titan/pipeline.py

```diff
@@ -0,0 +1,30 @@
+"""
+Pipelines for Titan
+"""
+
+from openedx_filters import PipelineStep
+
+from commerce_coordinator.apps.titan.clients import TitanAPIClient
+
+
+class GetTitanOrders(PipelineStep):
+    """
+    Adds titan orders to the order data list.
+    """
+
+    def run_filter(self, params, order_data):  # pylint: disable=arguments-differ
+        """
+        Execute a filter with the signature specified.
+        Arguments:
+            params: arguments passed through from the original order history url querystring
+            order_data: any preliminary orders (from earlier pipeline step) we want to append to
+        """
+
+        titan_api_client = TitanAPIClient()
+        titan_response = titan_api_client.create_order(**params)
```

Comment:
We are not calling the correct `create_order` here.

There should be a new `create_order` implement as part of the story [THES-136](https://2u-internal.atlassian.net/browse/THES-136)
Seem like that method is missing from this PR.

updated create_order method should call 3 Titan's APIs as shown in this [Mermaid](https://mermaid.live/edit#pako:eNrNVl1v2jAU_SvIz-A0kKTgh0kboKkPUyuK-jBFirz4AlYTO7MdWob63-eYpd2qAkajUvPkXPt86NyrxFuUSwaIoEUhH_IVVaYz_5IKXf9YKlqtOteKgepMwFBe6FR07MO4gtxwKdzJppLLsgSVQ5ZLqRgX1EiFaVVpDO0W1nwpaKGxbAizXAE1wLJdtdPrfTpAYrihYh8BXYO_iVpZgpUxlSZB0FZ7fwOAPVqFZfCMCZweIV-n8yM2X2TWHB40dtHd2aW_v1dAb8FD4R5TL0qNF7wwoP4ojx18Bj9r0JbGqzkVr6DgAvAOPG-KjsxH3b8r9rR_PxrqEzvxBsRD5FB676O4Q7SRVXRTgjBtSoTMppOr2XQ8Pya-611ecIvW2DXt883V2L1jyljGDZRe_W85WktLMLq0HxNQ2EID6yygFQ_WYZDbatByE3Jzfft_Ni2sKsBA5ubi3GZXkN_L2gStyjkMuwHJmhzeIdrzGTw9zz0D9CHM_DsmH8PSyyD4-dn7kT2DixNi2f8j9vJhqL5_C5w1G34ODlKcLwwQDHWR5SkpZ_aKtG3cpcisoIQUEbtksKB1YVKUiid7lNZG3m5EjsjCBgRdVFfM8k04tZep8lV1yrg19VysqEBkix4R6Q1jHMVRPw6jfn94MQjDLtogEvdxEo2iOAkHyeVwGF5GT130S0rLG-L4Ypgkg9EoDpOkH8UDx_fdbRpVW3pwat92dz135Xv6DcSGq8E)

As we are doing [here](https://github.com/edx/commerce-coordinator/blob/563d09dd4828282a3ce071f5250a21b32b8a60b4/commerce_coordinator/apps/titan/tasks.py#L36) in `order_created_save_task`.
Then `order_created_save_task` should be updated to use this new order_create.

existing `order_create` should be renamed to `create_cart` (see diagram)

Reply:
I think (according to tests) performs as expected. Im working on a final validation in the next day.